### PR TITLE
Fixes #6495: KeyError: 'old_value'

### PR DESCRIPTION
**Pull request by peckpeck**
http://www.rudder-project.org/redmine/issues/6495

### CODE REVIEW

**Comment by peckpeck** [comment]
PR updated

**Comment by peckpeck** [comment]
PR updated

**Comment by peckpeck** [comment]
PR updated

**Comment by peckpeck** [comment]
PR updated

File: scripts/rudder-dev/rudder-dev [hunk 1]
```diff
@@ -262,39 +262,49 @@ def get_ticket_info(ticket_id):
...
+      ticket_info['custom_fields'] = [ { 'id': CUSTOM_FIELD_PR, 'value': pr_url } ]
   else:
-    ticket_info = {
-            'issue': {
-                'notes': note,
-            }
-        }
+    if note is None:
+      return
+    else:
```
Comment (VinceMacBuche) [inline scripts/rudder-dev/rudder-dev]: What happens here with a non-developper ? I don't understand what will be the result with that empty ticket_info object!

**Comment by VinceMacBuche** [comment]
It's globally correct to me, but i need more explanation on some part!

File: scripts/rudder-dev/rudder-dev (hunk 1 above)
Comment (peckpeck) [inline scripts/rudder-dev/rudder-dev]: Indeed, I forgot to re add notes for a non developer

**Comment by peckpeck** [comment]
PR updated

**Comment by VinceMacBuche** [comment]
Alright! Merging !!

**Pull request merged**